frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -753,6 +753,7 @@ export class TileViewPage extends React.Component {
         onShowAllToggle={() => this.onShowAllToggle(groupName)}
         showAll={_.get(filterGroupsShowAll, groupName, false)}
         data-test-group-name={groupName}
+        showText={Object.keys(filterGroup).length - 5 + showMore}
```

Comment:
A few thing things here...

1. Magic numbers are typically very bad for applications; they add ambiguity to the code which just slows down maintenance cycles and can potentially lead to a bug if the number is mis-represented
2. The number is a PF preset for the prop `maxShowCount` -- we don't want to assume PF will never change this; let us define it in a variable, pass it into PF & use it to do our calculation on the number of keys in `filterGroup`
3. Concatenating strings is almost never going to work for i18n code; not all languages perceive the order of words in the same fashion as others -- and in order for us to do proper translations, we gotta call `t('i18n text string', { variableName: variableValue })` ... but there is a bit of a hiccup here. The text is given to us, we don't (as `TileViewPage`) know what that _variableName_ would be... I think this leads us to solving the problem with:
    1.  Moving the translation text into this component
    2. Calling the `t` function as mentioned above
    3. Since the translation is *needed* -- we don't even need to provide a prop to enable it; if we don't translate it, PF stuffs an English string and that's not good for an i18n enabled app.